hugr-passes/src/lib.rs

```diff
@@ -17,10 +17,34 @@ pub use inline_funcs::inline_acyclic;
 pub mod linearize_array;
 pub use linearize_array::LinearizeArrayPass;
 pub mod lower;
-pub mod merge_bbs;
 mod monomorphize;
+pub mod normalize_cfgs;
 pub mod untuple;
 
+/// Merge basic blocks. Subset of [normalize_cfgs], use the latter.
+#[deprecated(note = "Use normalize_cfgs", since = "0.15.1")]
```

Comment:
Mixed up the version with tket's
```suggestion
#[deprecated(note = "Use normalize_cfgs", since = "0.23.0")]
```

Reply:
lol, should have twigged that was low...15 was ages ago! This looks ahead/predicts the next number, right...